tools/helpers.cmake

```diff
@@ -297,7 +297,6 @@ endfunction(config_option)
 # configuration headers.
 macro(config_set optionname configname value)
     set(${optionname} "${value}" CACHE INTERNAL "" FORCE)
-    set(c_define "CONFIG_${configname}")
```

Comment:
As `config_set` is a macro, a caller can use `c_define` to get name of the config in C as a return value.

Reply:
(That being said, that probably wasn't the intention of the original variable assignment), but some downstream caller could be using it for this purpose by now.

Reply:
That is true, but I have doubts this is really the case anywhere. It was added by accident, as it was just necessary in some intermediate development state as helper variable. So I'd prefer to drop this to avoid building legacy. It was never documented either.
Nevertheless, if there is a use case, it might be interesting to learn about that. We publish things as JSON and YAML now that handle all this, so these should be used.

Reply:
It is true that this has only ever been available in the current devel version, it was added after the most recent release. According the GitHub search function, it is not used anywhere in the seL4 org. If there is downstream code that uses it, it at least never had any guarantee of stability. So I'd be fine with removing it.